CHANGES.md

```diff
@@ -28,9 +28,17 @@ this file to understand what changed.
 
 - new features
   - table for list of available cohorts
+  - added more metrics to ErrorAnalysisDashboard, including precision, recall and mean abs error
 - bug fixes
   - fix various UI usability bugs
   - fix for clicking custom scatter plot point
+  - ## Error Analysis
```

Comment:
The `##` might format this as very large in some markdown viewers

Reply:
hmm, in 0.10 release we used same thing for different components, it's as large as the v0.11.0 section header